apps/dashboard/app/javascript/xdmod/jobs.js

```diff
@@ -77,15 +79,16 @@ function table(context, helpers) {
   // <table class="table table-sm table-striped table-condensed">
   tableElement.classList.add('table', 'table-sm', 'table-striped', 'table-condensed');
 
-  thead = document.createElement('thead');
+  const thead = document.createElement('thead');
+  // Empty th to accommodate for the job analytics button
   thead.innerHTML = '<tr> \
+                      <th></th> \
```

Comment:
This should have a heading. You can use the class `sr-only`, but I think actual text here could be nice too (if it's a bit short/succinct).

Reply:
will add some text